brambling/views/organizer.py

```diff
@@ -1249,8 +1298,10 @@ def get_forms(self):
                         form.data = self.request.POST
                         form.is_bound = True
                         break
-        if show_payment_form:
-            forms = [self.payment_form, self.notes_form]
+        self.transaction_refund_forms = [TransactionRefundForm(t)
```

Comment:
It looks like there's no provision here for TransactionRefundForm to be instantiated with `request.POST` data, i.e. for displaying previously-submitted values. Just wanted to double-check that this is intentional.